tests/bsim/bluetooth/host/iso/cis/src/cis_central.c

```diff
@@ -237,7 +240,8 @@ static void reconfigure_cig(void)
 	/* Add CIS to the CIG and restore interval to 10ms */
 	param.cis_channels = &channels[1];
 	param.num_cis = 1U;
-	param.interval = interval_us; /* us */
+	param.c_to_p_interval = interval_us; /* us */
+	param.p_to_c_interval = interval_us; /* us */
```

Comment:
We could consider adding a test that has different interval for the two if supported (ping @cvinayak is it?)

Reply:
yes, I am working on it

Reply:
done